docs/reference/modules/settings.md

```diff
@@ -0,0 +1,127 @@
+
+In addition to the required `fields` property, each object can take five additional optional properties.
+
+The `label` property takes a string to display to the user in both preview and edit mode. If not supplied, Apostrophe will use the label for the first schema passed to the `fields` property. 
+
+The `help` property takes a field to display to the user in preview mode. If this property is present, it will replace any component added through the option `previewComponent` property. The `previewComponent` property takes the name of a Vue component that has been added to your project through any `modules/my-custom-module/ui/apos/components` folder. The string passed to this property should not contain the file extension. [See below](#previewcomponent) for further information about the component structure.
+
+The `protection` property can take a value of either `true` or `"password"`. These values are equivalent and will require that the user enters their password when attempting to change the value(s) of this `fields` object. Additional fields that are added by default can be added through the `addProtectedField()` method of the module.
+
+Finally, the `reload` property defaults to false, but can take a value of `true` if the preview should update after the user makes a change. This isn't needed if the `help` property is set.
+
+### groups
+
+The `groups` property takes an object of named objects. Each of the individual objects takes two properties, `label` and `subforms`. The label is displayed as the left hand tab name in the Personal Settings menu. The `subforms` property takes an array of `subforms` object names as strings. These subforms will be added to the tab in the order that they appear in the array. If no groups property is added to the settings, the `subforms` objects will be added to a tab named "Ungrouped" in the order that they were created. Similarly, any `subforms` objects not explicitly added to a `groups` object will be added to the "Ungrouped" tab.
+
+### `previewComponent`
+
+The preview component is displayed to the right of a `subforms` label when the user first opens a tab. If not supplied, Apostrophe will attempt to create a preview by combining the values of any schema fields within the individual `subforms` object as a space-separated string. For example, if the object has two schema fields, `firstName` and `lastName`, the resulting preview would be the equivalent to a template with `{{ firstName }} {{ lastName }}`. While this would work well for an object with a small number of fields, this might not be optimal for an object with a larger number.
```

Comment:
Not quite right, it's `previewComponent` or else `preview` or else the automatic behavior you describe (and `help` plays some role which I have asked Miro to clarify).

Reply:
Changed.